### DISCUSSION TITLE
Rejections due to "1.1.2 Distinct Function & Value; Accurate Representation"

### DISCUSSION THREAD

**Discussion by gnilly** [microsoft/MicrosoftEdge-Extensions#116] [General]
I attempted to publish a new extension multiple times and received a generic response:
```
1.1.2 Distinct Function & Value; Accurate Representation
The product, including its listing, does not accurately represent the product and may mislead users. Your product may not use a name or icon that is too similar to that of other products.
```
Some time ago, it also mentioned something like "Note the title," which would help me identify the problem.

Here's what I've tried:
- Making random changes to the extension's metadata, but it didn't work.
- Appealing to [reportap@microsoft.com](mailto:reportap@microsoft.com) also resulted in a generic reply:
```
Hello,
Thank you for contacting us. The product, including its listing, does not accurately represent the product and may mislead users. Your product may not use a name or icon that is too similar to that of other products.  Products need to have unique functionality and value in the Store. You may want to review the metadata section of our Product Quality guidelines here: https://msdn.microsoft.com/en-us/library/windows/apps/mt652261.aspx.
Thank You,

Microsoft Store Team
```

**Comment by ManikanthMSFT** [comment]
Hey @gnilly, could you please confirm was your extension published or are you still facing the rejection while publishing it?

**Reply by gnilly** [reply]
I managed to get help from ext_dev_support@microsoft.com.  
It took about two weeks with multiple follow-ups, but I finally got a reply from a human being.  
This reply was helpful, and I was able to publish the extension.

Overall, the experience was frustrating.

**Reply by ManikanthMSFT** [reply]
@gnilly, thank you for writing back to us and sharing your experience with us. We sincerely apologize for the delay and the frustration it caused. Your feedback is invaluable, and we are committed to improving our support process to ensure a more efficient and satisfactory experience in the future.

We are glad to hear that you were able to publish your extension successfully with the help you received. If you have any further concerns or need assistance with anything else, please do not hesitate to contact us. We are here to provide you with the best service possible.

Thank you for your patience and understanding.
